### DISCUSSION TITLE
CSS styling not visible on GitHub pages

### DISCUSSION THREAD

**Discussion by KaranPhagurah** [community/community#114560] [New to GitHub]
Hello everyone, 

I am trying to practice CSS and made a little website for that. I have been writing the HTML and CSS code in the codespace provided by GitHub and then committing the changes. In the live server, I can see my CSS code working just fine but when i publish it using GitHub pages, HTML is working but CSS is not being applied. 

I have tried to look for solutions online and even on the GitHub discussion pages, but I think I might be missing something out as I am not able to figure what is wrong with it.

https://github.com/KaranPhagurah/Poster_Website

Thank you in advance for your inputs.

Karan

Live server screenshot
![image](https://github.com/community/community/assets/87574751/96122806-7d4b-402f-a0db-870923362998)

Published site screenshot
![image](https://github.com/community/community/assets/87574751/327c2976-7b81-4a9e-a10a-a12d07e90d93)

**Comment by Tgentil** [comment]
@KaranPhagurah , I forked your project and tested it on GitHub Pages. It looks like you've already fixed the path for your `style.css` in your latest commit:

[Commit fixing style.css path](https://github.com/KaranPhagurah/Poster_Website/commit/0ccca3bc904bfdb7f1b58c2b880e83d005230edf)

However, there's an issue with your image path:

```html
<img class="mountain-img" src="/Images/mountain.jpg" alt="mountain">
```

To fix this, simply adjust the path like so:

```html
<img class="mountain-img" src="./Images/mountain.jpg" alt="mountain">
```

This change should ensure your image loads correctly on GitHub Pages.

**My deploy of your project on GitHub Pages:**
![first image](https://github.com/community/community/assets/88681055/770fb199-24bf-4adb-a344-ee1fe0782b51)

**My deploy after ajusting the path of the image:**
![second image](https://github.com/community/community/assets/88681055/b138b11b-5650-4f7f-a3fd-93d496696fd9)
